dubbo-metrics/dubbo-metrics-annotation/pom.xml

```diff
@@ -23,20 +23,18 @@
     <version>${revision}</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <artifactId>dubbo-metrics-event</artifactId>
+  <artifactId>dubbo-metrics-annotation</artifactId>
```

Comment:
Why should it be separated from the dubbo-metrics-api?

Reply:
The new commit separates MetricsServiceExporter to dubbo-metrics-api, and rename dubbo-metrics-annotation to dubbo-metrics-monitor.
dubbo-metrics-monitor retains some monitor interfaces. These interfaces have been marked as deprecated and are still relied upon by various packages.
I'm not sure how to deal with them, do you have any suggestions?